.travis.yml

```diff
@@ -31,8 +31,7 @@ matrix:
 before_install:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then which cmake || brew update        ; fi
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then which cmake || brew install cmake ; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew upgrade cmake libpng libtiff || true; fi
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install libtool openjpeg xz ; fi # depends for ImageMagick 7
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew upgrade cmake || true; fi
```

Comment:
The `xcode9` image has new enough cmake, so all this brew update, install and upgrade calls can be removed.